test/e2e/azure_suite_test.go

```diff
@@ -32,6 +32,7 @@ import (
 	. "github.com/onsi/ginkgo"
 	"github.com/onsi/ginkgo/config"
 	"github.com/onsi/ginkgo/reporters"
+	"github.com/onsi/gomega"
```

Comment:
gomega is already imported below as a `.` import

Reply:
@cpanato can you please fix this?

Reply:
done